### PR TITLE
Use a local copy of ReLinker

### LINKED ISSUE #2415: Gradle: Cannot find Relinker
Hi,
I am getting error when i try to use 0.88.0 instead of 0.87.5

`Error:Failed to resolve: com.github.KeepSafe:ReLinker:1.1`

### PR DESCRIPTION
**Pull request by emanuelez**
The problem is that libraries that use Realm were forced to tell users to add the jitpack repo manually, which is inconvenient.

Fixes #2415

### CODE REVIEW

**Comment by emanuelez** [comment]
@realm/java

**Comment by cmelchior** [comment]
The changelog for 0.88.1 needs to be updated, apart from that :+1:

**Comment by beeender** [comment]
:+1:

**Comment by emanuelez** [comment]
Changelog:
 * Fixed an issue resolving ReLinker in apps using a library based on Realm

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,8 +4,9 @@
 
 * Prevent throwing NullPointerException in RealmConfiguration.equals(RealmConfiguration) when RxJava is not in the classpath (#2416).
 * RealmTransformer fails because of missing annotation classes in user's project (#2413).
-* add SONAME header to shared libraries (#2432).
+* Added SONAME header to shared libraries (#2432).
 * now DynamicRealmObject.toString() correctly shows null value as "null" and the format is aligned to the String from typed RealmObject (#2439).
+* Fixed an issue occurring while resolving ReLinker in apps using a library based on Realm (#2415)
```
Comment (kneth) [inline CHANGELOG.md]: Please add `.` at the end.

**Comment by kneth** [comment]
:+1:

**Comment by zoonooz** [comment]
:+1: Thanks !

**Comment by jcj52436999** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  jcjacksonengineering@gmail.com
    Domain joe-jackson.net has exceeded the max emails per hour (116/100 (115%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext6.iad.github.net ([192.30.252.197]:38371 helo=github-smtp2b-ext-cp1-prd.iad.github.net)
	by rs5.websitehostserver.net with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.86_1)
	(envelope-from <noreply@github.com>)
	id 1aflk5-001g7L-Qv
	for jcjackson@joe-jackson.net; Tue, 15 Mar 2016 05:58:21 -0400
Date: Mon, 14 Mar 2016 04:41:53 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1457955713;
	bh=SodcbQulmtsn+O6RMU0rX8xEU7G0GNK2mvctYIeDmps=;
	h=From:Reply-To:To:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=I+byABM6XznBIYyQbU2vMFt+c37R+IWe1g5lGJcbWIPPSDblLI47vo25MoAr45xoM
	 vGhbETqQtOqye8LzTxEAB//GhVcScXfyswO3YtcL39ZAtPiY5olVtUnKgXwlWZ+FWW
	 OnNT/hxk84BZurpHmGtDQQySy34reknwW9ptrPx4=

**Comment by jcj52436999** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  jcjacksonengineering@gmail.com
    Domain joe-jackson.net has exceeded the max emails per hour (120/100 (120%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext2.iad.github.net ([192.30.252.193]:54289 helo=github-smtp2b-ext-cp1-prd.iad.github.net)
	by rs5.websitehostserver.net with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.86_1)
	(envelope-from <noreply@github.com>)
	id 1afmNf-001xzQ-NA
	for jcjackson@joe-jackson.net; Tue, 15 Mar 2016 06:39:15 -0400
Date: Mon, 14 Mar 2016 04:11:39 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1457953899;
	bh=54Hugbt++2ydf6oxzvjXzMONs+d2hVSWo0AyCs56eEU=;
	h=From:Reply-To:To:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=TI5Wj+UcPQh1I6Ym7uwX4VlJV5XxaI6k3JiqBgEVJMOoSr80kf7Nb2bLvFcTH4+hj
	 /TJUAT3gbhr1HAtjlyUUstwwgNukl+0ym7yR/5NmDRbwO2UvAdvTafnQWmTpKZ2AIy
	 H+epU/Cwbhmh1o1zpW/1q8oxsKJA6MslYAtD9AEY=

**Pull request merged**